src/nodekit.ts

```diff
@@ -61,12 +61,14 @@ export class NodeKit {
             appInstallation,
             appEnv,
             appDevMode,
+            appLoggingLevel: process.env.APP_LOGGING_LEVEL || fileConfig.appLoggingLevel,
```

Comment:
I think we should call this `appDefaultLoggingLevel` - in case we'll add additional methods for logging on different levels